### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by tensafefrogs** [googleads/google-media-framework-ios#60]
Lets do an accessibility audit: Fire up the iOS screen reader and make sure the player is (The whole demo app should be) accessible.

**Comment by shawnbuso** [comment]
I gave a quick pass with the accessibility inspector and the test app looks pretty good. If anyone is having accessibility issues with the player or app let us know and we'll get it fixed.
